src/main/java/com/scalar/cassy/service/MetadataDbBackupService.java

```diff
@@ -63,7 +63,7 @@ private Path sqliteBackup(Path file) {
   }
 
   private Path getKey(Path file) {
-    SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+    SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd_HH:mm:ss");
```

Comment:
I added a `_` character to avoid having a blank space in the file name

Reply:
Is it valid format ?
Before I experienced it throws an exception if it is invalid format.

Reply:
I was able able to upload files using this pattern so it seems valid.